documentation/gtfs_to_ntfs_specs.md

```diff
@@ -9,14 +9,14 @@ specification, the conversion should stop immediately with an error, unless
 otherwise specified.
 
 At the end of the conversion, a sanitizing operation is started on the final
-model. See [common.md](common.md) for more information.
+model. See [common NTFS specifications] for more information.
```

Comment:
This is not really "common NTFS specifications" since _specifications_ are on [another Github repository](https://github.com/CanalTP/ntfs-specification/). This is more automatic processing and rules we apply every time we're producing an NTFS: sanitizing dangling ID, reconstructing missing route names, recalculating stop area geolocation, etc. Maybe "common NTFS operations", or "common NTFS rules" would be more appropriated.

Reply:
What about "common operations during conversion" (along with "common_conversion_operations.md") ?
I think that we should more highlight the fact that it is about general rules applied _when converting_ an input dataset into NTFS than NTFS itself.

Reply:
By the way, is it always for conversion from a format into NTFS? For example, ˋ ˋrestrict-validity-period` is not such a conversion but is probably applying the rules too.

Reply:
@pbougue spotted this too and you're right, it's not always a conversion. So, highlighting it might not be a good idea after all :)
I would go for "common NTFS rules".